script/test_2.py

```diff
@@ -0,0 +1,28 @@
+import re
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+  
+
+def main():
+  driver = webdriver.Chrome(executable_path="bin/chromedriver")
+  driver.get("https://en.wikipedia.org/wiki/Main_Page")
+  search_input = driver.find_element_by_id("searchInput")
+  search_input.clear()
+  search_input.send_keys("Unit Testing")
+  search_input.submit()
+  if "Unit Testing" in driver.current_url:
+    print("We are on a good site")
```

Comment:
this is not needed. It was ok before

Reply:
I don't know how to make it work. When I wrote this:
```
 assert "Unit Testing" in driver.title
AssertionError
```
or I import unittest:
```
assertIn("Unit Testing", driver.title)
NameError: name 'assertIn' is not defined
```
it works only with if statement :(

Reply:
```python
assert "Unit Testing" in driver.title
```
Should be ok, what error do you get?

You have the same line in test_1.py and it works.
Maybe the problem is that the text is not in the title?

Reply:
it  is in the title: `<title> Unit testing - Wikipedia </title>`
And that's why I don't understand it.
I only have "AssertionError"

Reply:
Notice that it is written in different case :-)
If you don't care about letter cases than convert it to lower case first.

Reply:
in fact, I didn't pay attention to it. I'm sorry

Reply:
Nah. Not a problem. This can happen to anyone. :-)